### ISSUE TITLE
Confusion about the training

### ISSUE THREAD

**Issue by gancx** [XinhaoMei/WavCaps#31]
Thanks a lot for your effort on open-sourcing the dataset and training code. I have several questions related to this repo. 

1. I found there are several empty audio files (0 seconds) in the training set, which are not excluded from the training stage. Examples are {"id": "YVcu0pVF1npM.wav", "caption": "Male singing and music can be heard.", "audio": "wav_path", "duration": 0.0} {"id": "Y06-g5jz-OGc.wav", "caption": "Machine gun fire is heard.", "audio": "wav_path", "duration": 0.0} and etc. I wonder whether it will be removed during training.

2. Do you have the statistics, like the number of training samples, after excluding audios from AudioCaps and Clotho for zero-shot retrieval? On my side, the number is 388791.

3. After pertaining, the fine-tuning is done on Clotho and AudioCaps, respectively, or you did the fine-tuning at the same time. It seems that the best model saved on the Clotho validation set, then it will be fine-tuned on the Clotho training set. Another best model saved on AudioCaps will be fine-tuned on the AudioCaps training set. I wonder if my interpretation is correct.

**Comment by XinhaoMei** [comment]
Hi, sorry for my late reply.

1. They were not removed if the audios can be loaded by librosa. 

2. I cannot remember it exactly, but I think you could find this number in the paper. 

3. The fine-tuning is done on Clotho and AudioCaps, respectively.
